server/v2/cometbft/utils.go

```diff
@@ -344,13 +353,8 @@ func (c *consensus[T]) validateFinalizeBlockHeight(req *abci.FinalizeBlockReques
 
 // GetConsensusParams makes a query to the consensus module in order to get the latest consensus
 // parameters from committed state
-func (c *consensus[T]) GetConsensusParams(ctx context.Context) (*cmtproto.ConsensusParams, error) {
-	latestVersion, err := c.store.GetLatestVersion()
-	if err != nil {
-		return nil, err
-	}
-
-	res, err := c.app.Query(ctx, latestVersion, &types.QueryParamsRequest{})
+func GetConsensusParams[T transaction.Tx](ctx context.Context, app handlers.AppManager[T]) (*cmtproto.ConsensusParams, error) {
```

Comment:
Why did this need to be refactored?

The use of `c.GetConsensusParams` is a lot clearer than the new syntax introduced

Reply:
I think because it is used in `ValidateVoteExtensions` and that is something you'd be calling in the app directly.

Reply:
Yeah, it's also a nice helper to have. I don't think it's much more complicated than what we had